src/components/AppBar/AppBar.jsx

```diff
@@ -176,19 +197,26 @@ const AppBar = ({
   );
 
   return (
-    <>
-      <Box className={classes.root} data-testid="app-bar">
-        <MaterialAppBar position="fixed" className={classes.appBar}>
-          <Toolbar>
-            {!isLoading ? (<Logo />) : null}
-            <Box className={classes.navLinkBar}>
-              <AddASpace />
-              {userContext.userProfile.username ? <PositionedMenu /> : <LogIn />}
-            </Box>
+    <Box className={isDesktopWidth || !showSearchBar ? classes.root : [classes.root, classes.expandAppBar]} data-testid="app-bar">
+      <MaterialAppBar position="fixed" className={[classes.appBar, classes.muiBreakpointWorkaround]}>
+        <Toolbar className={classes.muiBreakpointWorkaround}>
+          {!isLoading ? (
+            <Logo />
+          ) : null}
+          {(isDesktopWidth && showSearchBar)
+            ? <Box className={classes.expandedContent}><SearchBar /></Box> : null}
+          <Box className={classes.navLinkBar}>
+            <AddASpace />
+            {userContext.userProfile.username ? <LoggedInDropdown /> : <LogIn />}
```

Comment:
Is it possible for `userProfile` to be null ? might want to use `userContext.userProfile?.username`

Reply:
Not sure if userProfile can be null unless we change the implementation but I went forward with the changes and added a .eslintrc rule to prevent errors with optional chaining.